mne_bids/mne_bids.py

```diff
@@ -325,10 +386,20 @@ def _coordsystem_json(raw, unit, orient, manufacturer, fname, verbose):
         Used to define the coordinate system for the MEG sensors.
     fname : str
         Filename to save the coordsystem.json to.
+    write_mode : str, one of ('overwrite', 'error')
```

Comment:
missing `'append'`

Reply:
you will never use `'append'` here ... they are subject-level files